### DISCUSSION TITLE
Is there any suggested hosting provider ?

### DISCUSSION THREAD

**Discussion by octalpixel** [quirrel-dev/quirrel#944] [Q&A]
Wondering is there a suggested hosting provider for the   Quirrel, that has a decent free tier.

Anyone has prior experience ?

**Comment by 0xabrar** [comment]
Seems highly unlikely you're gonna be able to run this on free-tier software (Redis instances are usually pretty expensive). I'm currently running this on DigitalOcean, but I still need to figure out what the ideal machine specs/ideal configuration are.
